README.md

```diff
@@ -72,7 +72,14 @@ $ iex --sname c -S mix
 iex(c@mymachine)> {:ok, _pid} = Raft.start_peer(KVStore, name: :peer3)
 ```
 
-At this point our peers are started but currently they're all in the
+Raft currently prints a lot of debug info; to quiet this, you can
```

Comment:
I'm not sure that the excess logging that we do is useful for most people. We could probably remove stuff like the heartbeat logs.

Reply:
I find it useful, and since it's logged at `:debug` level, it is out of the way most of the time unless you turn on debug logging for max verbosity anyway. You could make debug logging configurable though, so that things are still logged at that level, but only if something like `trace: true` is set in the config, that way you can snuff the logs if you really don't want them.